packages/plugin/src/theia-proposed.d.ts

```diff
@@ -35,20 +35,20 @@ declare module '@theia/plugin' {
         export function stop(id: string): void;
     }
 
-    // Experimental API
-    // https://github.com/Microsoft/vscode/blob/1.30.2/src/vs/vscode.proposed.d.ts#L1015
-    export interface FileRenameEvent {
-        readonly oldUri: Uri;
-        readonly newUri: Uri;
-    }
-
-    // Experimental API
-    // https://github.com/Microsoft/vscode/blob/1.30.2/src/vs/vscode.proposed.d.ts#L1020
-    export interface FileWillRenameEvent {
-        readonly oldUri: Uri;
-        readonly newUri: Uri;
-        waitUntil(thenable: PromiseLike<WorkspaceEdit>): void;
-    }
+    // // Experimental API
+    // // https://github.com/Microsoft/vscode/blob/1.30.2/src/vs/vscode.proposed.d.ts#L1015
+    // export interface FileRenameEvent {
+    //     readonly oldUri: Uri;
+    //     readonly newUri: Uri;
+    // }
+
+    // // Experimental API
+    // // https://github.com/Microsoft/vscode/blob/1.30.2/src/vs/vscode.proposed.d.ts#L1020
+    // export interface FileWillRenameEvent {
+    //     readonly oldUri: Uri;
+    //     readonly newUri: Uri;
+    //     waitUntil(thenable: PromiseLike<WorkspaceEdit>): void;
+    // }
```

Comment:
Should I keep these apis or remove them?

Reply:
- generally we should remove commented out code there is git history
- if code is moved to theia.d.ts from proposed then it should be removed from proposed